src/components/VideoTrack/VideoTrack.tsx

```diff
@@ -35,7 +35,8 @@ export default function VideoTrack({ track, isLocal, priority }: VideoTrackProps
   }, [track, priority]);
 
   // The local video track is mirrored.
-  const style = isLocal ? { transform: 'rotateY(180deg)' } : {};
+  const isFrontFacing = track.mediaStreamTrack.getSettings().facingMode !== 'environment';
```

Comment:
How about `track.mediaStreamTrack.getSettings().facingMode === 'user';`? Because if facingMode is `left` or `right`, this evaluates to true.

Reply:
`facingMode` is a property that seems to exist only on mobile devices. So `isFrontFacing` would be `false` when using the app on my laptop, and the video would not be mirrored when it should be. 

Also, according to MDN, facingMode of `left` and `right` represents cameras that are front facing, just pointed over the user's left or right shoulder. I can't think of any devices with these cameras, but we would want `isFrontFacing` to be `true` in those cases.